browser/brave_shields/https_upgrade_browsertest.cc

```diff
@@ -30,7 +30,7 @@
 #if !BUILDFLAG(IS_ANDROID)
 #include "chrome/browser/ssl/ask_before_http_dialog_controller.h"
 #include "chrome/browser/ui/browser_navigator_params.h"
-#include "chrome/browser/ui/tabs/public/tab_features.h"  // nogncheck
+#include "chrome/browser/ui/tabs/public/tab_features.h"
```

Comment:
see https://github.com/brave/brave-core/pull/32111#discussion_r2623890916 cpp and gn guards should always match so this should move out of `!BUILDFLAG(IS_ANDROID)`